067-kraft-jbod-support.md

```diff
@@ -0,0 +1,223 @@
+
+### JBOD usage in controller-only nodes
+
+Controller-only nodes host only the metadata log.
+They do not host any regular partition replicas.
+The metadata log is always placed in a single directory / volume.
+As such, there is no real value of using multiple volumes in controller-only nodes in terms of any performance improvements.
+But using multiple volumes for a short period of time might be used to address various situations such as changing the type of the volume used for the metadata etc.
+Therefore, Strimzi will not prevent users from configuring multiple volumes for controller-only nodes.
+But it will issue a warning when multiple volumes are specified for controller-only nodes.
+
+### Dedicated metadata value
+
+On brokers or on Kafka nodes with mixed roles, the volume / directory used for the metadata log will be used to also host regular partition-replicas.
+In some situations, this might not be desired as the metadata log might compete with the regular Kafka logs for storage or performance.
+Technically, it is possible to use a different data volume for the metadata log and for the regular Kafka logs.
+However, it is not expected that this would be a common configuration because:
+* Clusters sensitive to performance are expected to use dedicated controller-only nodes.
+  And on the controller-only nodes, there are no regular partition-replicas.
+  So the metadata log has its own dedicated volume even without any special support for it.
+* On broker-only nodes, the impact of shared volumes for data and metadata is expected to be small.
+  And using a dedicated volume just for a metadata is not expected to be very efficient in terms of the utilization of the dedicated disk.
+
+Based on the above, there will be no support for dedicated metadata log volumes.
+This should help us to save unnecessary effort on the development and testing of this feature.
+
+However, if needed, the support for it can be added later if we see the demand for it:
+* Update the API and add a new valid value `dedicated` to the `kraftMetadata` field in the storage definition
+* Configure this volume only as the `metadata.log.dir` and do not use it in `log.dirs`
+* Add a validation to prevent changing of an existing volume used for regular logs to metadata-only log without all partition-replicas being moved to other volumes first.
+
+### Example YAML files and documentation
+
+The examples YAML files will be adapted to cover the API changes:
+* A new KRaft JBOD example will be added with multiple disks and the `kraftMetadata` used for one of them
+* The existing KRaft example with a single persistent volume will be adapted to add the `kraftMetadata: shared` option to the custom resource
+
+The documentation would suggest to Strimzi users to mark one of the volumes for being used for the metadata as that should help to make it more clear where the metadata are stored and might also help while helping users etc.
```

Comment:
```suggestion
The documentation would recommend that Strimzi users designate one of the volumes specifically for storing metadata. 
This would make the storage location for metadata clear, and might help with tasks such as maintenance.
```